--- libiconv-1.15/libcharset/lib/localcharset.c.ok	2018-04-22 10:39:53.000000000 +0800
+++ libiconv-1.15/libcharset/lib/localcharset.c	2018-04-22 10:47:13.000000000 +0800
@@ -123,7 +123,6 @@
 static const char *
 get_charset_aliases (void)
 {
-  int c;
   const char *cp;
 
   cp = charset_aliases;
@@ -193,6 +192,7 @@
 
                   for (;;)
                     {
+                      int c;
                       char buf1[50+1];
                       char buf2[50+1];
                       size_t l1, l2;
